src/filesystem/CMakeLists.txt

```diff
@@ -19,6 +21,7 @@ add_library(smf_filesystem STATIC
 
 target_link_libraries(smf_filesystem
   smf_utils
+  xxhash
```

Comment:
with `-DXXH_PRIVATE_API` we no longer need xxhash as a library. All the calls get inline now.

Reply:
if I didn't do it on master it is a bug, i can fix it and delete the libxxhash_static cmake rule.

Reply:
Ah thanks. I didn't look to see how xxhash was being used, but linking the library solved the include path issue. I'll fix this up and just deal with include paths.